CHANGELOG.rst

```diff
@@ -26,7 +26,7 @@ Changes:
 
 - Empty strings are valid events now.
   `#110 <https://github.com/hynek/structlog/issues/110>`_
-
+- ``structlog.dev.ConsoleRenderer`` now handles log events with all-caps log-levels, in addition to the default lowercase levels generated by ``structlog.stdlib.add_log_level``
```

Comment:
Please

- add () behind add_log_level (it’s nicer to read if you can identify functions/callables on one blink)
- end the sentence with a period
- add a link to this pr in a separate line